Utilities/src/main/java/edu/stanford/hivdb/utilities/Sequence.java

```diff
@@ -67,20 +66,29 @@ public Sequence(String header, String sequence) {
 		this.sequence = sanitizeSequence(sequence);
 	}
 
-	public static List<Sequence> fromGenbank(Collection<String> accessions) {
-		return FastaUtils.fetchGenbank(accessions);
-	}
-
+//	public static List<Sequence> fromGenbank(Collection<String> accessions) {
+//		return FastaUtils.fetchGenbank(accessions);
+//	}
+	
 	public static Sequence fromGenbank(String accession) {
-		List<String> accessions = new ArrayList<>();
-		accessions.add(accession);
-		List<Sequence> result = FastaUtils.fetchGenbank(accessions);
-		if (result.isEmpty()) {
-			return null;
+		List<String> accessions = Arrays.asList(accession);
+		List<Sequence> result;
+		try {
+			result = FastaUtils.fetchGenbank(accessions);
+		} catch (RuntimeException e) {
+			throw new RuntimeException(e);
 		}
+		if (result.isEmpty() || result.get(0).isEmpty()) {
+			return null;
+		} 
 		return result.get(0);
 	}
-
+	
+	private boolean isEmpty() {
+		if (this.header.equals("UnamedSequence")) return true;
```

Comment:
Oo, I see. Thank you for those tips. 

I found that, when it receives a malformed query, the server returns:

`"  Error: CEFetchPApplication::proxy_stream(): Bad id. Must be 'accession' or 'gi' or 'gi:sat:satkey'"`

So I turned `resultStr` to an empty string in this case in `FastaUtils.cleanup`

Regarding the exception I mentioned, it seems to be a client error (443). So it could be specific to the environment. I'll remove the try/catch block so we can reach 100% coverage in Sequence.